src/GOATalgorithm.jl

```diff
@@ -0,0 +1,152 @@
+#flat doubly stochastic matrix, starting point
+"""
+    _flat_doubly_stochastic(n)
+
+Given the dimension 'n', return the flat doubly stochastic matrix with number of rows and columns equals to 'n'.
+The flat doubly stochastic matrix is the barycenter of doubly stochastic matrices.
```

Comment:
It would be better to just write here in $\LaTeX$ that it is the matrix $\frac 1n \mathbb 1^T \mathbb 1$.